src/main/java/com/pirimid/cryptotrade/helper/exchange/EXCHANGE.java

```diff
@@ -0,0 +1,13 @@
+package com.pirimid.cryptotrade.helper.exchange;
+
+public enum EXCHANGE {
+    AAX("AAX"),
+    COINBASE("Coinbase"),
+    GEMINI("Gemini"),
+    KUCOIN("Kucoin");
```

Comment:
can we remove AAX and Kucoin for now?

Reply:
Yes, If we are not using it then you can.

Reply:
okay